repository/f.json

```diff
@@ -43,6 +43,16 @@
 				}
 			]
 		},
+		{
+			"name": "FabricEngine",
+			"details": "https://github.com/stepcg/SublimeFabricEngineSyntax",
```

Comment:
Please add the "language syntax" label to make your package easier to find, and help people understand exactly what it is.